shell/main.go

```diff
@@ -73,46 +69,37 @@ func (n *notifee) HandlePeerFound(p peer.AddrInfo) {
 
 func main() {
```

Comment:
This "shell" can eventually run an API server like the daemon, and provide a kind of model playground. In the meantime, I moved it to the root dir (along with daemon) and left the chat app in there.